### ISSUE TITLE
RangeError: Invalid count value

### ISSUE THREAD

**Issue by arichiardi** [anmonteiro/lumo#92]
Hey Antonio, I have just got the following error:

```
Lumo 1.2.0
ClojureScript 1.9.482
 Docs: (doc function-name-here)
 Exit: Control+D or :cljs/quit or exit

Lumo socket REPL listening at localhost:5044.
cljs.user=> [eval]:2
!function e(t,r,n){function o(s,a){if(!r[s]){if(!t[s]){var u="function"==typeof require&&require;if(!a&&u)return u(s,!0);if(i)return i(s,!0);var l=new Error("Cannot find module '"+s+"'");throw l.code="MODULE_NOT_FOUND",l}var f=r[s]={exports:{}};t[s][0].call(f.exports,function(e){var r=t[s][1][e];return o(r?r:e)},f,f.exports,e,t,r,n)}return r[s].exports}for(var i="function"==typeof require&&require,s=0;s<n.length;s++)o(n[s]);return o}({1:[function(e,t,r){(function(t,n,o,i,s,a,u,l){function f(e){return new Error("getopt: "+e)}function c(e,t,r){var n;for(d(e||""===e,"optstring is required"),d(e.constructor===String,"optstring must be a string"),d(t,"argv is required"),d(t.constructor===Array,"argv must be an array"),this.gop_argv=new Array(t.length),this.gop_options={},this.gop_aliases={},this.gop_optind=void 0!==r?r:2,this.gop_subind=0,n=0;n<t.length;n++)d(t[n].constructor===String,"argv must be string array"),this.gop_argv[n]=t[n];this.parseOptstr(e)}var d=e("assert").ok;r.BasicParser=c,c.prototype.

RangeError: Invalid count value
    at String.repeat (native)
    at y ([eval]:2:19309)
    at Interface.<anonymous> ([eval]:2:25036)
    at emitOne (events.js:96:13)
    at Interface.emit (events.js:189:7)
    at Interface._onLine (readline.js:238:10)
    at Interface.<anonymous> (readline.js:375:12)
    at Array.forEach (native)
    at Interface._normalWrite (readline.js:374:11)
    at Socket.ondata (readline.js:98:10)
                              java.lang.Thread.run                       Thread.java:  745
java.util.concurrent.ThreadPoolExecutor$Worker.run           ThreadPoolExecutor.java:  617
 java.util.concurrent.ThreadPoolExecutor.runWorker           ThreadPoolExecutor.java: 1142
               java.util.concurrent.FutureTask.run                   FutureTask.java:  266
                                               ...                                        
               clojure.core/binding-conveyor-fn/fn                          core.clj: 2020
                                 boot.core/boot/fn                          core.clj: 1029
                               boot.core/run-tasks                          core.clj: 1019
                        boot.user/eval430/fn/fn/fn  boot.user2787845353405966573.clj:   37
                        boot.user/eval469/fn/fn/fn  boot.user2787845353405966573.clj:   39
                                               ...                                        
                                    boot.util/dosh                          util.clj:  411
       java.lang.Exception: lumo: non-zero exit status (1)
clojure.lang.ExceptionInfo: lumo: non-zero exit status (1)
    file: "/tmp/boot.user2787845353405966573.clj"
    line: 43
```

**Comment by arichiardi** [comment]
I am trying to isolate the form that breaks.

**Comment by daemianmack** [comment]
I'm seeing this error whenever I tell Emacs' `inf-clojure` to evaluate a nested form containing newlines.

```
;; Bad.
(comment
  (defn foo []

    :one))
```
Issuing `inf-clojure-eval-last-sexp` against that form kills Lumo. 

Removing the embedded newline or pulling it out of the `comment` form makes it safe to eval again.

```
;; Good.
(comment
  (defn foo []
    :one) )
```
```
;; Good.
  (defn foo []

    :one) 
```

This is the clearest thing I've been able to isolate, but I've seen the issue with more interesting forms, like...

```
;; Bad.
(defmethod parse-event :FooEvent [event]
  (let [message (cond
                  (pos? 1) :pos

                  (neg? 1) :neg)]))
```
```
;; Good.
(defmethod parse-event :FooEvent [event]
  (let [message (cond
                  (pos? 1) :pos
                  (neg? 1) :neg)]))
```

**Comment by anmonteiro** [comment]
fixed in #120
